test/numerical/TestConv.cpp

```diff
@@ -29,20 +27,19 @@ bool isOMConvTheSameAsNaiveImplFor(const int N, const int C, const int H,
     const int W, const int kH, const int kW, int pHBegin, int pHEnd,
     int pWBegin, int pWEnd, const ConvAutoPad autoPad) {
   static int testNum = 0;
-  if (DEBUG)
-    printf(
-        "attempt %d with N %d, C %d, H %d, W %d, kH %d, kW %d, pHBegin %d, "
-        "pHEnd %d, pWBegin %d, pWEnd %d, autopad %s, isDynamic %d, stride %d, "
-        "dilation %d\n",
-        ++testNum, N, C, H, W, kH, kW, pHBegin, pHEnd, pWBegin, pWEnd,
-        Conv2DLibBuilder::getAutoPadName(autoPad).c_str(), isDynamic, stride,
-        dilation);
+  printf("attempt %d with N %d, C %d, H %d, W %d, kH %d, kW %d, pHBegin %d, "
```

Comment:
Are you sure you don't want to keep this printf only under DEBUG?

Reply:
I removed it for consistency with [other tests](https://github.com/onnx/onnx-mlir/tree/main/test/numerical) The other tests don't have `DEBUG`. Even without `DEBUG`, the print statement is not output when running numerical tests using cmake command. To print it out, we need to add `--verbose` in [this line](https://github.com/onnx/onnx-mlir/blob/main/test/numerical/CMakeLists.txt#L8).

Reply:
I will add `DEBUG` in all tests if requested.